core/src/main/java/com/netflix/iceberg/BaseTableScan.java

```diff
@@ -41,6 +42,7 @@
 import com.netflix.iceberg.util.ParallelIterable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+
```

Comment:
Please remove unnecessary whitespace additions.

Here's an example problem caused by this: https://github.com/apache/spark/commit/6d22ee3969597ed5e38333d5b6f0f891a859594e#r32770189

Reply:
My bad. I'll be more careful next time.

Reply:
Not a problem, I just wanted to highlight that these do actually cause problems later on so it's best to catch them during reviews.